app/assets/stylesheets/vendor/main.css

```diff
@@ -3462,6 +3462,16 @@ div.timeline-year {
     transition: 0.3s ease;
 }
 
+.map article.widthS {
+    width: 100%;
+    margin-right: -34.1799%;
```

Comment:
This CSS is applied in the default map view.  It is important to make the width of this element 100% rather than 65.8201% so the full map renders.  If the map renders at 65.8201% when the page loads, then you will not see an expanded map when you hide the facets - rather, you will see a block with part of the map and the rest filled up with grey.  By rendering the map at 100% when the page loads, and "hiding" part of the map in the negative margin, it ensures that the you can see the the entire map when you hide the facets without having to reload the map (which takes a long time).